Pipfile

```
@@ -8,10 +8,10 @@ argo-workflows = ">=3.5,<4"
 daiquiri = "*"
 requests = "*"
 sentry-sdk = "*"
-kubernetes = "~=11.0.0"  # required pyyaml>=3.12
-openshift = "==0.11.0"
+kubernetes = ">=12.0.1"  # required pyyaml>=5.3.1
+openshift = "==0.12.0"
```

```suggestion
openshift = "~=0.12.0"
```
How about this https://github.com/openshift/openshift-restclient-python/tree/v0.12.1
It has some changes: https://github.com/openshift/openshift-restclient-python/commits/v0.12.1